common/rfb/SSecurityPlain.cxx

```diff
@@ -36,7 +36,8 @@ using namespace rfb;
 
 StringParameter PasswordValidator::plainUsers
 ("PlainUsers",
- "Users permitted to access via Plain security type (including TLSPlain, X509Plain etc.)",
+ "Users permitted to access via Plain security type (including TLSPlain, X509Plain etc.) "
+ "or RSA-AES security types (RA2, RA2ne, RA2_256, RA2ne_256)",
```

Comment:
Not a huge fan of these names, and having so many of them. But it seems to be how RealVNC are naming things, so I guess we shouldn't come up with something different.